tensorflow_lite_support/cc/task/processor/image_preprocessor.cc

```diff
@@ -91,7 +110,7 @@ absl::Status ImagePreprocessor::Preprocess(const FrameBuffer& frame_buffer,
   std::unique_ptr<FrameBuffer> preprocessed_frame_buffer;
   std::vector<uint8> preprocessed_data;
 
-  if (IsImagePreprocessingNeeded(frame_buffer, roi)) {
+  if (IsImagePreprocessingNeeded(frame_buffer, roi) && !IsInputImageDynamic()) {
```

Comment:
What is an image needs preprocessing (cropping, rotation) while the input shape is dynamic?

Reply:
For rotation, I think for all outputs kTopLeft is fixed right? So for ```preprocesed_frame```,  I need to set input_specs_.width = roi.width similarly for height. The orientation can remain the same (kTopLeft).

Reply:
output_buffer kTopLeft is fixed right, but you'll need to reorient the output_buffer to the same orientation of the input image. See the [example of image_segmenter](https://github.com/tensorflow/tflite-support/blob/ba16516f0f06558a38201958253b42e14c913447/tensorflow_lite_support/cc/task/vision/image_segmenter.cc#L396-L397).

I need to set input_specs_.width = roi.width similarly for height => Sounds good!

Reply:
As discussed in meeting, kTopLeft is set for ```output_buffer```. Resolving this.